GameDefinition/game-src/Client/UI/Content/Input.hs

```diff
@@ -63,12 +63,18 @@ standardKeysAndMouse = InputContentRaw $ map evalKeyDef $
               , "clear messages and show history"
               , ExecuteIfClear LastHistory ))
   , ("Tab", ( [CmdMove]
```

Comment:
I'd put `CmdMinimal` only here, of all 4 variants. Currently the 'Minimal cheat sheet' menu overflow.s